portal-api/src/v1/gef/mappings/map-companies-house-data.js

```diff
@@ -0,0 +1,30 @@
+const mapCompaniesHouseData = (data, industries) => {
+  const address = data.registered_office_address;
+
+  let selectedIndustry = null;
+  if (industries && industries.length === 1) {
+    // eslint-disable-next-line prefer-destructuring
+    selectedIndustry = industries[0];
```

Comment:
can we do `[selectedIndustry] = industries;`? - and we can remove the `//eslint-disable...` comment from above